File: clients/csharp/Bots/Bomb.cs
using System.Text.Json.Serialization;

namespace CsClient.Bots
{
    public class Bomb
    {
        [JsonPropertyName("row")] public int Row { get; set; }

        [JsonPropertyName("col")] public int Col { get; set; }

        [JsonPropertyName("explode_in_round")] public int ExplodeInRound { get; set; }

        [JsonConstructor]
        public Bomb(int row, int col, int explodeInRound)
        {
            Row = row;
            Col = col;
            ExplodeInRound = explodeInRound;
        }
    }
}

File: clients/dart/model/bomb.dart
class Bomb {
  final int row;
  final int col;
  final int explode_in_round;

  Bomb({required this.row, required this.col, required this.explode_in_round});

  factory Bomb.fromJson(Map<String, dynamic> json) {
    return Bomb(
      row: json['row'] as int,
      col: json['col'] as int,
      explode_in_round: json['explode_in_round'] as int,
    );
  }
}
